recipes/libflint/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+source activate "${CONDA_DEFAULT_ENV}"
```

Comment:
We should be able to drop this now as we are using a newer `conda-build` on OS X.